lib/components/opaque_image_view.js

```diff
@@ -5,28 +5,55 @@ import React from 'react'
 import {
   processColor,
   requireNativeComponent,
-  ColorPropType,
   PixelRatio,
   StyleSheet
 } from 'react-native'
 
 import colors from '../../data/colors'
+import type { LayoutEvent } from '../system/events'
 
 const GeminiHost = 'd7hftxdivxxvm.cloudfront.net'
 const ImageQuality = 85
```

Comment:
This turned out to be a bit wrong, we do need to keep the Proptypes around, not sure of the right way to merge the two ATM, so calling that for another day